packages/authenticator/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -227,10 +227,16 @@ class StateMachineBloc
           yield UnauthenticatedState.confirmSignInNewPassword;
         case AuthSignInStep.confirmSignInWithTotpMfaCode:
           yield UnauthenticatedState.confirmSignInWithTotpMfaCode;
+        case AuthSignInStep.confirmSignInWithEmailMfaCode:
+          yield UnauthenticatedState.confirmSignInWithEmailMfaCode;
         case AuthSignInStep.continueSignInWithMfaSelection:
           yield ContinueSignInWithMfaSelection(
             allowedMfaTypes: result.nextStep.allowedMfaTypes,
           );
+        case AuthSignInStep.continueSignInWithMfaSetupSelection:
+          yield ContinueSignInWithMfaSetupSelection(
+            allowedMfaTypes: result.nextStep.allowedMfaTypes,
+          );
         case AuthSignInStep.continueSignInWithTotpSetup:
```

Comment:
Q: do we need a case for `continueSignInWithEmailMfaSetup` here?

Reply:
Yep, this is in the next PR for authenticator changes